config/i18n-tasks.yml

```diff
@@ -85,6 +85,12 @@ search:
 
 ## Do not consider these keys missing:
 ignore_missing:
+  - datetime.dotiw.days
```

Comment:
Why do we need these here in `ignore_missing`?

Reply:
We don't have the French translations yet.